.github/workflows/ci.yml

```diff
@@ -140,9 +140,6 @@ jobs:
           service_account_key: ${{ secrets.GCP_SA_KEY }}
           export_default_credentials: true
 
-      - name: Error on warning
-        run: echo "RUSTFLAGS=-D warnings" >> $GITHUB_ENV
```

Comment:
Alternatively, we could set `CARGO_BUILD_RUSTFLAGS` or pass it to `cargo build` directly. However, I don't think there's any situation where `cargo build` would catch something `lint.js` wouldn't.